esphome/components/airthings_wave_plus/sensor.py

```diff
@@ -73,3 +82,5 @@ async def to_code(config):
     if config_illuminance := config.get(CONF_ILLUMINANCE):
         sens = await sensor.new_sensor(config_illuminance)
         cg.add(var.set_illuminance(sens))
+    if config_device_type := config.get(CONF_DEVICE_TYPE):
+        cg.add(var.set_device_type(config_device_type))
```

Comment:
has a default will always be defined
```suggestion
    cg.add(var.set_device_type(config[CONF_DEVICE_TYPE])
```